### DISCUSSION TITLE
Political Playoff Simulation

### DISCUSSION THREAD

**Discussion by MarkStair** [bytemaster/clarion#21] [Ideas]
Political playoff idea from Dan is intriguing, however as far as I know the process has never been tested.
I propose a political playoff "simulation" to test how it works out.  Here are my thoughts on how to do a quick and easy simulation:

1. Political Playoffs to select the Clarion logo.
2. The clarion discussion group currently has about 22 members plus @bytemaster.  These could be randomly grouped into about 6 groups of 4 or 4 groups of 6 members.  I feel that @bytemaster should oversee the process rather than be a member of any group, but that's up to Dan.
3. Each group selects One representative per the political playoff (PP) mechanism
4. All the representatives get together and select ONE Logo and anything else proposed during their "term".
5. Dan can review the logo selected and approve or veto.  This is not part of the PP mechanism, however, if Dan approves the logo selected we can determine the test "positive" and if he does not choose the PP selected logo, we can determine that the PP process was a "negative" for this particular experiment.

**Comment by jonathanbahai** [comment]
I think the end results to determine positive or negative outcomes to suit the personal preferences of Dan is a weak indicator.

I would be more interested in the measures of timeliness of organization of groups, and time it took to reach consensus, and how it all actually performed. 

No negative or positive rating really, more like a baseline experiment to help define expectations of performance between the groups etc.

**Reply by MarkStair** [reply]
Yeah sure.  The main point is to test the PP process.  Maybe the PP reps can come to consensus of what worked and didn't work as expected.

**Reply by bytemaster** [reply]
I believe that this is a fantastic idea. I would probably consider allowing everyone in the community to participate in choosing someone to propose a logo. 

I don't think the experiment could be judged by whether I agree with the winner. But the outcome would provide me ample feedback on what resonates and how the process works.

**Reply by brandonlovejoy** [reply]
I love it. This is a great idea. :) When logos are ready for consideration, we should find a date and give enough notice for optimal turnout. Determining which person can best represent consensus around something aesthetic through this process is sure to be weirdly abstract, and intriguing.
